public/app/plugins/panel/geomap/components/DataHoverView.tsx

```diff
@@ -59,56 +62,92 @@ export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode }:
   }
 
   return (
-    <table className={styles.infoWrap}>
-      <tbody>
-        {(mode === TooltipDisplayMode.Multi || mode == null) &&
-          displayValues.map((v, i) => (
-            <tr key={`${i}/${rowIndex}`} className={i === columnIndex ? styles.highlight : ''}>
-              <th>{v[0]}:</th>
-              <td>{v[2]}</td>
+    <div className={styles.wrapper}>
+      <div className={styles.header}>
+        <span className={styles.title}>Exemplar</span>
+      </div>
+      <table className={styles.infoWrap}>
+        <tbody>
+          {(mode === TooltipDisplayMode.Multi || mode == null) &&
+            displayValues.map((v, i) => (
+              <tr key={`${i}/${rowIndex}`} className={i === columnIndex ? styles.highlight : ''}>
+                <th>{v[0]}:</th>
+                <td>{v[2]}</td>
+              </tr>
+            ))}
+          {mode === TooltipDisplayMode.Single && columnIndex && (
+            <tr key={`${columnIndex}/${rowIndex}`}>
+              <th>{displayValues[columnIndex][0]}:</th>
+              <td>{displayValues[columnIndex][2]}</td>
             </tr>
-          ))}
-        {mode === TooltipDisplayMode.Single && columnIndex && (
-          <tr key={`${columnIndex}/${rowIndex}`}>
-            <th>{displayValues[columnIndex][0]}:</th>
-            <td>{displayValues[columnIndex][2]}</td>
-          </tr>
-        )}
-        {links.length > 0 && (
-          <tr>
-            <td colSpan={2}>
-              <VerticalGroup>
-                {links.map((link, i) => (
-                  <LinkButton
-                    key={i}
-                    icon={'external-link-alt'}
-                    target={link.target}
-                    href={link.href}
-                    onClick={link.onClick}
-                    fill="text"
-                    style={{ width: '100%' }}
-                  >
-                    {link.title}
-                  </LinkButton>
-                ))}
-              </VerticalGroup>
-            </td>
-          </tr>
-        )}
-      </tbody>
-    </table>
+          )}
+          {links.length > 0 && (
+            <tr>
+              <td colSpan={2}>
+                <VerticalGroup>
+                  {links.map((link, i) => (
+                    <LinkButton
+                      key={i}
+                      icon={'external-link-alt'}
+                      target={link.target}
+                      href={link.href}
+                      onClick={link.onClick}
+                      fill="text"
+                      style={{ width: '100%' }}
+                    >
+                      {link.title}
+                    </LinkButton>
+                  ))}
+                </VerticalGroup>
+              </td>
+            </tr>
+          )}
+        </tbody>
+      </table>
+    </div>
   );
 };
 
-const getStyles = (theme: GrafanaTheme2) => ({
-  infoWrap: css`
-    padding: 8px;
-    th {
+const getStyles = (theme: GrafanaTheme2) => {
```

Comment:
We will want to verify this doesn't cause any regressions for the geomap tooltip (and any other places this datahoverview is being used as well)

Reply:
We want to unify the exemplar popover component UI in all views actually. I did not pay attention it is coming from geomap. I can make a shared component and import it into other chart views. 
For the verification, I only add a couple more styles. I do not think it will cause any regressions. Is there anything I can do to verify that?

Reply:
This branch:
<img width="261" alt="image" src="https://user-images.githubusercontent.com/109082771/206512257-19940f10-4f16-4c64-b2fd-32f074756c2a.png">
<img width="282" alt="image" src="https://user-images.githubusercontent.com/109082771/206512311-2707ab45-8b79-4d83-a0f0-ffe20b3b1c97.png">
<img width="238" alt="image" src="https://user-images.githubusercontent.com/109082771/206512403-687022da-8d6e-4e90-9d6b-0435c8f8e9af.png">
<img width="791" alt="image" src="https://user-images.githubusercontent.com/109082771/206512536-51c45b74-7cb6-4764-8f1a-1474745279fa.png">

Main:
<img width="226" alt="image" src="https://user-images.githubusercontent.com/109082771/206512784-0f27426b-6a56-4631-8340-adb776ff46a4.png">
<img width="266" alt="image" src="https://user-images.githubusercontent.com/109082771/206512852-b71cf8b0-84f4-46d9-803d-f088392af7e7.png">
<img width="785" alt="image" src="https://user-images.githubusercontent.com/109082771/206512886-ab73fdbd-cf7c-4189-b519-3b2d228af0b9.png">


The only change I'm seeing between the implementations for geomap is that the "exemplar" header is visible on this branch, but not on main

Reply:
So I think we make the "exemplar" header conditional (so it's only being rendered when we are expecting it to be there (in heatmap for example) or make it opt out if we want it everywhere except for geomap tooltip (via a prop))